src/c-writer.cc

```diff
@@ -94,8 +94,9 @@ struct GlobalName {
   const std::string& name;
 };
 
-struct ExternalPtr : GlobalName {
-  using GlobalName::GlobalName;
+struct TagPtr : GlobalName {
+  explicit TagPtr(const std::string& name)
```

Comment:
is `Ptr` the right name here?

Reply:
Welll... yes technically (which is why it was ExternalPtr before -- the tags are represented as a `void*`), but we can probably find a clearer name. `TagSymbol`?

Reply:
Renamed to `TagSymbol`